docs/en/02_Developer_Guides/09_Security/00_Member.md

```diff
@@ -127,6 +127,13 @@ things, you should add appropriate `[api:Permission::checkMember()]` calls to th
 	  }
 	}
 
+## Saved user logins ##
```

Comment:
Hah, I wrote that guideline as well ... serves me right. But this specific case is inconsistent either way - sentence case would mean that "logins" is lowercase.